WordPress/Sniffs/PHP/DiscouragedPHPFunctionsSniff.php

```diff
@@ -97,6 +97,16 @@ public function getGroups() {
 					'str_rot13',
 				),
 			),
+			'typecasting' => array(
+				'type'      => 'warning',
+				'message'   => '%s() can be replaced with type casting.',
+				'functions' => array(
+					'intval',
+					'strval',
+					'floatval',
+					'boolval',
```

Comment:
What about `doubleval()` and `settype()` ?

Reply:
Added in 2aeb355